src/components/Quote.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import { FaCircleNotch, FaQuoteLeft } from 'react-icons/fa';
+
+function Quotes() {
+  const [data, setData] = useState([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const apiKey = 'zWbFdXe3BQWrlhFsnImSAw==HEDkeSdVW05kaUH7';
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to use (`.env`) environment variables to store the API key safely and not expose it in your code